apps/web-giddh/src/app/ledger/ledger.component.html

```diff
@@ -294,14 +300,18 @@ <h3 class="ledgerHead">
                                                 <table class="width100">
                                                     <tbody class="ledger-main">
                                                         <tr class="ledger-row">
-                                                            <td class="col-xs-1 text-center">
-                                                                <input type="checkbox" [(ngModel)]="txn.isChecked" (click)="selectEntryForBulkAction($event, item)" />
-                                                            </td>
-                                                            <td class="col-xs-2">
-                                                                <input class="form-control" [maxlength]="10" name="bsDatess" autocomplete="off" type="text" #cbs='bsDatepicker' [(ngModel)]="item.entryDate" bsDatepicker [bsConfig]="{ dateInputFormat: giddhDateFormat }" [outsideClick]="true">
-
+                                                            <td class="col-xs-3 col-md-3 col-lg-3">
+                                                                <div class="d-flex align-items-center">
+                                                                    <span>
+                                                                        <input type="checkbox" [(ngModel)]="txn.isChecked" (click)="selectEntryForBulkAction($event, item)" />
+                                                                    </span>
+                                                                    <span class="ml-05">
+                                                                        <input class="form-control" [maxlength]="10" name="bsDatess" autocomplete="off" type="text" #cbs='bsDatepicker' [(ngModel)]="item.entryDate" bsDatepicker [bsConfig]="{ dateInputFormat: giddhDateFormat }" [outsideClick]="true">
```

Comment:
Avoid incorrect spelling in ** bsDatess**

Reply:
I am not added this, but i correct it